packages/react/src/file-upload/FileUpload.tsx

```diff
@@ -310,16 +417,16 @@ export const FileUpload = forwardRef<HTMLInputElement, FileUploadProps>(
 	}
 );
 
-const fileInputStyles = ({
+function fileInputStyles({
```

Comment:
What's the reason for this change?

Reply:
Essentially code style.

I use const if it fits on 1 line
```
const x = ( ) => y
```

or if it returns on one line
```
const x = () => ({
  a: 'b'
})
```

of if it is part of a callback
```
onClick={() => { 
  setA(true)
  setB(false)
}}
```

Otherwise I prefer using the `function` keyword because it makes it clearer that the variable is a function and not a value.

I also like that function encapsulates the full contents of the function so if a `const` goes to 2 lines, I prefer to convert it to a function
```
// This erks me since the funciton is not encapsulated
const x = ( ) => 
   y
    .one()
    .two()
```
```
// This erks me because it is less clear that x is a function than if the "function" keyword was used
const x = ( ) => {
   return y
    .one()
    .two()
}
```